### PR TITLE
Add `computer` (GitHub Codespaces) icon

### PR DESCRIPTION
**Pull request by danielbayley**
`computer` is [`codespaces`](https://primer.style/design/foundations/icons/codespaces-16), and `pilot` is [`copilot`](https://primer.style/design/foundations/icons/copilot-16) from octicons…

Alts:

Artboard 1@2x
<br/>
Untitled

Edit: Leaving final (co)`pilot` icon here, for reference…

preview

### CODE REVIEW

File: icons/pilot.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  stroke-linejoin="round"
+>
+  <path d="M10 15v2" />
+  <path d="M14 15v2" />
+  <path d="M8 11H7c-1.7 0-3-1.3-3-3V7c0-2.2 1.8-4 4-4h1c1.7 0 3 1.3 3 3v1c0 2.2-1.8 4-4 4z" />
+  <path d="M17 11h-1c-2.2 0-4-1.8-4-4V6c0-1.7 1.3-3 3-3h1c2.2 0 4 1.8 4 4v1c0 1.7-1.3 3-3 3z" />
+  <path d="M18 18h2a2 2 0 0 0 2-2v-3a2 2 0 0 0-2-2h-2v6a4 4 0 0 1-4 4h-4a4 4 0 0 1-4-4v-6H4a2 2 0 0 0-2 2v3a2 2 0 0 0 2 2h2" />
+  <path d="M5 18v4" />
+  <path d="M19 18v4" />
+</svg>
```
Comment (jguddas) [inline icons/pilot.svg:19]: Maybe something with small eyes for the sake of cuteness? icon (49)

Comment (danielbayley) [reply icons/pilot.svg:19]: @jguddas Options…

Artboard 1@2x
<br/>
Untitled

Comment (danielbayley) [reply icons/pilot.svg]: @jguddas Implemented…

**Comment by karsa-mistmere** [comment]
Computer doesn't quite work with this aspect ratio on the display.

For pilot can you try smaller goggle size and the same eyes as in `cat` and `dog`?

File: icons/pilot.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="8" cy="7" r="4" />
+  <circle cx="16" cy="7" r="4" />
+  <path d="M4 22a2 2 0 0 0 2-2v-2H4a2 2 0 0 1-2-2v-3a2 2 0 0 1 2-2h2v5a4 4 0 0 0 4 4h4a4 4 0 0 0 4-4v-5h2a2 2 0 0 1 2 2v3a2 2 0 0 1-2 2h-2v2a2 2 0 0 0 2 2" />
+  <path d="M10 15h.01" />
+  <path d="M14 15h.01" />
```
Comment (jguddas) [inline icons/pilot.svg:16]: Cute, to have everything connect, maybe try something like this.
```suggestion
  <circle cx="16" cy="7" r="4" />
  <circle cx="8" cy="7" r="4" />
  <path d="M4 22a2 2 0 0 0 2-2v-9.5" />
  <path d="M6 18H4a2 2 0 0 1-2-2v-3a2 2 0 0 1 2-2h2" />
  <path d="M6 16a4 4 0 0 0 4 4h4a4 4 0 0 0 4-4" />
  <path d="M18 11h2a2 2 0 0 1 2 2v3a2 2 0 0 1-2 2h-2" />
  <path d="M18 10.5V20a2 2 0 0 0 2 2" />
  <path d="M 10 15h0.01" />
  <path d="M 14 15h0.01" />
```

**Comment by danielbayley** [comment]
> Computer doesn't quite work with this aspect ratio on the display.

> For pilot can you try smaller goggle size and the same eyes as in `cat` and `dog`?

@karsa-mistmere Done and done…

**Comment by ericfennis** [comment]
I think the Github copilot icon is a brand icon, so I don't think we should add this one.

**Comment by jguddas** [comment]
> I think the Github copilot icon is a brand icon, so I don't think we should add this one.

It's just a `pilot` nothing GitHub related about it.

**Comment by ericfennis** [comment]
I'm closing this one. Due to weak use case.
@jguddas Yes it is a pilot icon. But other than copilot I don't see any other use case. So this one is too specific and brand related.

**Comment by jguddas** [comment]
Well, it made it into Unicode 🧑‍✈️, can't be that useless 🤷‍♂️

**Comment by danielbayley** [comment]
@ericfennis What about `computer`?

**Comment by ericfennis** [comment]
@danielbayley that one has good use cases.
So feel free to open up another PR

**Comment by danielbayley** [comment]
> @danielbayley that one has good use cases. So feel free to open up another PR

@ericfennis Wouldn't it just be easier to reopen this one, and I can just revert `pilot`…?

**Comment by ericfennis** [comment]
@danielbayley sure!

**Comment by karsa-mistmere** [comment]
I still think computer is not very recognisable as it currently is, because of the funky aspect ratio.

I'm not quite sure what specific use case it could cover that `laptop`, `laptop-2` and `pc-case` cannot.

**Comment by danielbayley** [comment]
> funky aspect ratio.

I think the current proportions work better…

> I’m not quite sure …

`pc-case` is vertical, and has no paired monitor. Also, it’s good enough for Octicons… https://primer.style/design/foundations/icons/codespaces-16

> sure!

@ericfennis Done.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**